### PR TITLE
Github Action tests

### PR DESCRIPTION
**Pull request by bderusha**

### CODE REVIEW

**Review by fredsmith — APPROVED** [review]
do you want to kill the travis tests in the same PR?

**Review by fredsmith — APPROVED** [review]

**Review by coryleeio — APPROVED** [review]

**Comment by edx-deployment** [comment]
GitHub Actions has started building this code into a docker file.  Check status at https://github.com/edx/edx-notes-api/actions/runs/366890309

**Comment by edx-deployment** [comment]
A docker container including this PR has been built and shipped to docker hub.  Check it out at https://hub.docker.com/r/openedx/edx-notes-api/tags?page=1&name=bb3bbf4d2956099d149a71d5c05e4290aa02c712

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        django-version: ['django22', 'django30']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Start container
+      run: |
+        docker-compose -f .travis/docker-compose-travis.yml up -d
+    - name: Install Dependencies
+      run: |
+        pip install -r requirements/travis.txt
```
Comment (adzuci) [inline .github/workflows/ci.yml:26]: Would be cool if we renames the travis.txt file.

**Pull request merged**